zebra: Coverity issue Medium null pointer dereference

hnattamaisub opened the pull request:
This commit addresses null pointer dereference in zebra folder.

CID covering:
109898, 72706, 90819, 109575

Description:
Handled null check accordingly.

mjstapp commented:
so ... please ask one of your colleagues about how to work with FRR PRs and git branches. it's really not correct to keep closing active PRs and opening new ones with similar/identical diffs. should I spend time with these changes again, or should I just ignore this PR?

hnattamaisub commented:
> > so ... please ask one of your colleagues about how to work with FRR PRs and git branches. it's really not correct to keep closing active PRs and opening new ones with similar/identical diffs. should I spend time with these changes again, or should I just ignore this PR?
> 
> sorry for this inconvenience , i will make sure not to open new PRs in future, i addressed your comments in new PR. The old PR , please ignore. Thanks and sorry again

let me reopen previous PR and make these changes there only after discussion, u can ignore this one
